functionsLab.js

```diff
@@ -0,0 +1,296 @@
+// # Function Exercises
```

Comment:
I see that you have good understanding of how functions work and scoping works. See feedbacks below and fix some issues when you have time. Other than that, good job and keep it up!